src/main/scala/org/dia/algorithms/mcc/GTGRunner.scala

```diff
@@ -287,17 +266,25 @@ class GTGRunner(val masterURL: String,
      * Note if no HDFS path is given, then randomly generated matrices are used.
      *
      */
-    val sRDD = sc.NetcdfDFSFiles(paths, List("ch4"), partitions)
+    val sRDD = sc.sciDatasets(paths, List("ch4", "longitude", "latitude"), partitions)
+
+    /**
+     * Collect lat and lon arrays
+     */
+    val sampleDataset = sRDD.take(1)(0)
+    val lon = sampleDataset("longitude").data()
+    val lat = sampleDataset("latitude").data()
 
     /**
      * Record the frame Number in each SciTensor
      */
     val labeled = recordFrameNumber(sRDD)
 
+
     /**
      * Filter for temperature values under 241.0
      */
-    val filtered = labeled.map(p => p("ch4") <= 241.0)
+    val filtered = labeled.map(p => p("ch4") = p("ch4") <= 241.0)
```

Comment:
The above line overwrites the original variable under key "ch4"
Do we want to preserve the original variable under key "ch4"?
In which case the expression would instead be something like

p("ch4masked") = p("ch4") <= 241.0